lib/kdump_utils.pm

```diff
@@ -27,10 +27,16 @@ our @EXPORT = qw(install_kernel_debuginfo prepare_for_kdump
 
 sub install_kernel_debuginfo {
     zypper_call 'ref';
-    # JeOS uses kernel-default-base, except on aarch64 openSUSE
-    my $kernel    = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
-    my $debuginfo = script_output('rpmquery --queryformat="%{NAME}-%{VERSION}-%{RELEASE}\n" ' . $kernel . '| sort --version-sort | tail -n 1');
-    $debuginfo =~ s/$kernel/$kernel-debuginfo/g;
+    my $kernel = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
+    my $debuginfo;
+    # there is no 'kernel-default-base-debuginfo' so use 'kernel-default-debuginfo' instead
+    if ($kernel eq 'kernel-default-base') {
```

Comment:
Not true, on SLE 12 and 15 pre-SP2 it's still `kernel-default-base-debuginfo`.

Reply:
The comment is wrong or incomplete, but isn't the case of `kernel-default-base-debuginfo` covered properly in the else branch?

Reply:
If `$kernel` equals `kernel-default-base` the else branch is never reached. There must be a check for the OS version.